superx/routes/home.py

```diff
@@ -32,21 +36,28 @@ def cart():
                 "name": item['name'],
                 "price": same_item.price
             })
-            
+
             total_prices[super_name]['price'] += same_item.price
 
     return render_template('cart.html', total_prices=total_prices)
 
 
 def livesearch():
+    '''
+    returns search functin using jquery data and ajax so not to redirect
+    '''
 
     json_list_of_items = []
 
     # if search_res is empty string, return empty json
     search_res = request.form.get("input")
     if not search_res:
         return render_template('products_table.html', products=json_list_of_items)
-    products_list = db.session.query(Product).order_by(Product.name).filter(Product.name.contains(search_res)).all()
+
+    #pylint: disable=no-member
```

Comment:
not sure why it writes that query has no member filter when it has so i disabled it